src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -156,6 +156,17 @@ internal class DefaultBuildScriptGenerator : IBuildScriptGenerator
                     }
 
                     platformVersion = detectionResult.LanguageVersion;
+                    _logger.LogDebug(
```

Comment:
Since `platformName` and `platformVersion` are standard names that we have been using all across our code base, it might become problematic to write queries just based on them. IMO we should start using `eventid` too. So it is like `get me traces where eventid=100 and platformName is dotnet` (here eventid 100 should represent the event of selecting a platform and version for the repo). Thoughts?

Reply:
cc @dorfire @gpcastro

Reply:
what is an event ID?

Reply:
It can uniquely identify a trace message all across an app. I believe it will also get logged as a dimension

Reply:
Example:
![image](https://user-images.githubusercontent.com/6559784/65078281-8a74ce80-d951-11e9-91d9-53c3d7465d99.png)

Reply:
those will very easily become confusing though; how are those event IDs generated? Is it AI that maintains them?

Reply:
No, these event IDs are maintained by us. We need to make sure we do not change them randomly so that our queries would not get broken. It requires discipline but I think it will make querying easy. Let's discuss this tomorrow and then I will merge it.